static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
+earlier versions or branches of the project directory, nor will it download
+files which have not changed.
+
+If the `--all-branches` or `--all-tags` options are specified, `dvc pull` will
+examine files associated with all branches or tags of the DVC files in
+the project directory.
+
+If one or more _target_'s are specified, DVC only considers the files associated
+with those stages.  Using the `--with-deps` option DVC tracks dependences
+backward through the pipeline to find files to pull.
+
+??? --force
+
+With the `--recursive dirname` option DVC searches the named directory and its
+subdirectories for files to download.
 
 After data file is in cache DVC utilizes OS specific mechanisms like reflinks or
 hardlinks to put it into the working space without copying. See `dvc checkout`
 for more details.
 
-See `dvc remote`, `dvc config` and
-[remote storages](https://dvc.org/doc/get-started/configure)
-for more information on how to configure the remote storage.
+## Options
 
-```usage
-    usage: dvc pull [-h] [-q] [-v] [-j JOBS] [-r REMOTE] [-a]
-                    [-T] [-d] [-f]
-                    [targets [targets ...]]
+* `--show-checksums` Show checksums instead of file names.
 
-    positional arguments:
-      targets               DVC files.
+* `-r REMOTE`, `--remote REMOTE` specifies which remote cache 
+  (see `dvc remote list`) to pull from. The argument, `REMOTE`, is a
+  remote name defined using the `dvc remote add` command.
 
-    optional arguments:
-      -h, --help            show this help message and exit
-      -q, --quiet           Be quiet.
-      -v, --verbose         Be verbose.
-      -j JOBS, --jobs JOBS  Number of jobs to run simultaneously.
-      --show-checksums      Show checksums instead of file names.
-      -r REMOTE, --remote REMOTE
-                            Remote repository to pull from
-      -a, --all-branches    Fetch cache for all branches.
-      -T, --all-tags        Fetch cache for all tags.
-      -d, --with-deps       Fetch cache for all dependencies of the specified
-                            target.
-      -f, --force           Do not prompt when removing working directory files.
-      -R RECURSIVE, --recursive RECURSIVE
-                            Pull cache for subdirectories of the specified
-                            directory.
-```
+* `-a`, `--all-branches`  Determines the files to download by comparing across
+  all branches of the workspace.
+
+* `-T`, `--all-tags` Determines the files to download by comparing across
+  all tags of the workspace.
+
+* `-d`, `--with-deps` Determines the files to download by searching backwards
+  in the pipeline from the named stage(s).  The only files which will be
+  considered are associated with the named stage, and the stages which execute
+  earlier in the pipeline.
+
+* `-f`, `--force` Do not prompt when removing working directory files.
+
+* `-R dirname`, `--recursive dirname` Determines the files to download by
```

Comment:
dirname -> DIRNAME like in other case if we actually mention them

Reply:
Yes